index.html

```diff
@@ -0,0 +1,196 @@
+
+  <body class="bg-light">
+    <header id="header">
+      <nav class="navbar navbar-expand-lg fixed-top navbar-light bg-light" id="nav-bar">
+        <div class="container">
+          <a class="navbar-brand" href="#">
+                <img src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" class="img-amazon" alt="" id="header-img">
+              </a>
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+              </button>
+          <div class="collapse navbar-collapse" id="navbarResponsive">
+            <ul class="navbar-nav ml-auto">
+              <li class="nav-item">
+                <a class="nav-link" href="#features">Features</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#works">How it works</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#pricing">Pricing</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <div class="row row-padding">
+      <div class="col-12">
+        <h4 class="alig">
+          <b>Handcrafted, home-made masterpieces</b>
+        </h4>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-4"></div>
+      <div class="col-4">
+        <form id="form" method="POST" action="https://www.freecodecamp.com/email-submit">
+          <div class="form-group">
+            <input type="email" class="form-control" name="email" id="email" placeholder="Enter email" required>
+          </div> <p></p><p></p>
+          <div class="form-group">
+            <div class="align-center">
+              <input type="submit" id="submit" class="btn btn-warning text-center"  value="GET STARTED"/>
+            </div>
+          </div>
+        </form>
+      </div>
+      <div class="col-4"></div>
+    </div> 
+    <div class="row top-padding">
+      <div class="row">
+          <div class="col-4 right-padding">
+            <i class="fas fa-fire ffire"></i>
+
+          </div>
+          <div class="col-7">
+              <div>
+                <h4><b> Premium Materials</b></h4>
+                Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.
+              </div>
+          </div>
+      </div> 
+      <a href="#features">.</a>
+      <div class="row padding-top-70" id="features">
+        <div class="col-4 right-padding-two">
+          <i class="fas fa-truck ffire"></i>
+        </div>
+        <div class="col-7">
+            <div>
+              <h4><b> 
+                Fast Shipping</b></h4>
+                We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied. 
```

Comment:
😇 This part can be wrapped nicely using the flexbox technique and the h4 tag. Why don't you give it a shot

Reply:
Good review comment you have here 👏🏻! You have acknowledged the good work made by the person you are reviewing. Now, let's make it better by giving them a reason why they should make the requested change!